perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -391,6 +392,13 @@ def _parse_arguments(argv):
       required=False,
       type=int,
   )
+  parser.add_argument(
+      '--log_file',
```

Comment:
Typical naming convention is `--log-file` for splitting words in python argparser. Any particular reason we're using `--log_file` instead ?

Reply:
This change was made to maintain consistency with the listing benchmark script. 
Since this argument is no longer in use, resolving this.